### PR TITLE
AlexNet paper

### PR DESCRIPTION
**Pull request by jodoin**

### CODE REVIEW

File: deep-learning/_posts/2017-03-01-alexnet.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+layout: review
+title: (ALEXnet) ImageNet Classification with Deep Convolutional Neural Networks
```
Comment (lemairecarl) [inline deep-learning/_posts/2017-03-01-alexnet.md]: ALEXnet --> AlexNet ?
(Alex n'est pas un acronyme)

File: deep-learning/_posts/2017-03-01-alexnet.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+tags: deep-learning CNN essentials
+cite:
+    authors: "A Krizhevsky, I. Sutskever, G. Hinton"
+    title:   "ImageNet Classification with Deep Convolutional Neural Networks"
+    venue:   "Proceedings of NIPS 2012, p.1097-1105"
+pdf:   "http://papers.nips.cc/paper/4824-imagenet-classification-with-deep-convolutional-neural-networks.pdf"
+---
+
+ 
+Super famous 2012 AlexNet paper which presented the first CNN applied to ImageNET.  5 conv layers with maxpooling and ReLU + 2 FC layers + softmax of 1000 classes.
```
Comment (lemairecarl) [inline deep-learning/_posts/2017-03-01-alexnet.md]: ImageNET --> ImageNet ?

**Comment by JavedSajid** [comment]
Are you guys applying this network architecture for image classification or some other application?

**Comment by Dref360** [comment]
@JavedSajid No, this is an old model that nobody uses today. But at that time, it was giving very good result. 

Modern nets like Xception or ResNet are wider and deeper. Also, they use new mechanisms that were not available when AlexNet was created.

**Comment by JavedSajid** [comment]
@Dref360 Yes I know this model very well. It is proposed in 2012 and got about 10,000 citations.

File: deep-learning/_posts/2017-03-01-alexnet.md (hunk 1 above)
Comment (jodoin) [inline deep-learning/_posts/2017-03-01-alexnet.md]: done

File: deep-learning/_posts/2017-03-01-alexnet.md (hunk 2 above)
Comment (jodoin) [inline deep-learning/_posts/2017-03-01-alexnet.md]: done

**Comment by jodoin** [comment]
I added this reference because I consider it as an "essential" paper that everyone shall read at least once.

**Pull request merged**